src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -107,46 +215,96 @@ function Push-KeyValueOutputBinding {
 .PARAMETER Force
     (Optional) If specified, will force the value to be set for a specified output binding.
 #>
-function Push-OutputBinding {
+function Push-OutputBinding
+{
     [CmdletBinding()]
     param (
-        [Parameter(
-            Mandatory=$true,
-            ParameterSetName="NameValue",
-            Position=0,
-            ValueFromPipelineByPropertyName=$true)]
-        [string]
-        $Name,
-
-        [Parameter(
-            Mandatory=$true,
-            ParameterSetName="NameValue",
-            Position=1,
-            ValueFromPipelineByPropertyName=$true)]
```

Comment:
I removed the `ValueFromPipelineByPropertyName` from both `-Name` and `-Value`, as well as the `-InputObject`.
My argument is that I don't think its current pipeline parameter binding behavior is useful in real scenarios. Currently it supports:
```
[pscustomobject] @{ Name = 'queue'; Value = 'xxx' } | Push-OutputBinding
@{ Name = 'queue'; Value = 'xxx' } | Push-OutputBinding
```
The proposed new behavior is:
```
<# produce values #> | Push-OutputBinding -Name queue
```